docs/source/contributing/community.md

```diff
@@ -4,7 +4,7 @@ We use different channels of communication for different purposes. Whichever one
 
 ## Discourse (recommended)
 
-We use [Discourse](https://discourse.jupyter.org) for online discussions and support questions. Everyone in the Jupyter community is welcome to bring ideas and questions there.
+We use [Discourse](https://discourse.jupyter.org) for online discussions and support questions. You can ask questions here if you are a first-time contributor to the jupyterhub project. Everyone in the Jupyter community is welcome to bring ideas and questions there.
```

Comment:
Looks like this branch includes #4166 . Can you rebase without that commit?

Reply:
hello @minrk I am not sure how to rebase. I have run the rebase command I am not sure if it fixed the issue. Please help me review it.

Reply:
Assuming you have `origin` as the canonical `jupyterhub/jupyterhub` repo (some folks use `upstream` for this):

```
# update origin
git fetch origin
# rebase
git rebase -i origin/main
```

This will open your editor with a file like:

```
pick 2ab515b1 Update to the services.md
pick 787d9de3 applied PR change suggestions
```

You can remove any lines with commits you don't want (they may be removed already by git because the other PR has been merged). Close the editor, and git will complete the rebase. You can then update your pull request:

```shell
git push -f
```

Reply:
Thank you so much. 🙂

Let me try this now.

Reply:
are you on the branch for this PR?

Reply:
I think I have rebased the branch, I switched to the PR branch

Reply:
Thank you. Please let me know  if I should do more changes.